src/ext-proxy.c

```diff
@@ -224,6 +224,11 @@ void ext_proxy_set_header(Client *c, const char *headers)
     dbus_call(c, "SetHeaderSetting", g_variant_new("(s)", headers), NULL);
 }
 
+void ext_proxy_focus_element_by_id(Client *c, const char *element_id)
+{
+    dbus_call(c, "FocusElementById", g_variant_new("(s)", element_id), NULL);
```

Comment:
We have to check if the call to the webextension was meant to the right pageId. And there is also a check in the webextension for that. So you need to give the pageId for the call `dbus_call(c, "FocusElementById", g_variant_new("(ts)", c->page_id, element_id), NULL);`